docs/dockercompose/11-discovery-configuration.md

```diff
@@ -0,0 +1,66 @@
+# Discovery configuration
+
+Discovery configuration is stored in the `discovery-config.yaml` file. This file has the following sections:
+
+```yaml
+enabled: 
```

Comment:
I'd rather put a table with parameters and descriptions instead of providing such a config, I think it can be confusing for the user. https://splunk.github.io/splunk-connect-for-snmp/main/microk8s/configuration/poller-configuration/#configure-inventory - maybe something like this? I think table + example configuration would be enough